python/utilities.py

```diff
@@ -97,6 +97,22 @@ def takeScreenshot(return_path_to_file=False):
         return fullImagePath
 
 
+def angleDegreesBetweenLatlons(latlon1, latlon2, referenceLatlon):
+    '''Calulates the angle between the XY coordinates of two latlons, given referenceLatlon located at (0,0)
+
+    Args:
+        latlon1 (sailbot_msg.msg._latlon.latlon): The first latlon to calculate the degree of
+        latlon2 (sailbot_msg.msg._latlon.latlon): The second latlon to calculate the degree of
+        referenceLatlon (sailbot_msg.msg._latlon.latlon): The latlon that will be located at (0, 0)
+
+    Returns:
+        float representing the absolute angle between the two latlons in degrees
+    '''
+    x1, y1 = latlonToXY(latlon1, referenceLatlon)
+    x2, y2 = latlonToXY(latlon2, referenceLatlon)
+    return abs(math.tan(y2 / x2) - math.tan(y1 / x1)) * 180 / math.pi
```

Comment:
I think that this has a couple of bugs:
1. tan(theta) = y/x (I think you need atan)
2. It is best to use atan2 instead of atan. atan can't tell the difference between 135 degrees (-1, 1) and -45 degrees (1, -1), atan2 can
3. Careful about wrap around. Minimal example below where raw angle diff is large but they are actually very close.
![image](https://user-images.githubusercontent.com/26510814/122699714-bc8a2300-d1fe-11eb-9099-3ff6579bb54d.png)

Reply:
Actually just realized we have a function for this. Look at `python/planner_helpers.py`

```
def abs_angle_dist_radians(angle1_radians, angle2_radians):
```

Reply:
Test it out on some simple examples to double check if works!

Reply:
the degrees version is now available

Reply:
Thanks for the help, I don't really know what I was thinking. Luckily there was an existing function to use!

Reply:
Just noticed the degrees version's output is still in radians. Is this the intended behaviour?

Reply:
No that's definitely a mistake by me haha. Could you fix that? :D